crates/core-new/src/cairo_runner/input/run.rs

```diff
@@ -0,0 +1,59 @@
+use cairo_lang_starknet_classes::casm_contract_class::CasmContractClass;
+use hdp_primitives::datalake::output::{Task, TaskFormatted};
+use hdp_provider::evm::AbstractProviderResult;
+use serde::Serialize;
+use serde_with::serde_as;
+use starknet::core::serde::unsigned_field_element::UfeHex;
+use starknet_crypto::FieldElement;
+
+/*
+    input.json file that will be passed to the processor, is generated by this struct.
+*/
+
+#[derive(Serialize)]
+pub struct RunnerInput {
+    /// Batched tasks root of all tasks.
+    pub task_root: String,
+    /// if every tasks are pre computable, this can be Some
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub result_root: Option<String>,
+    /// Fetched proofs per each fetch point.
+    pub proofs: AbstractProviderResult,
+    /// tasks compatible with v1
+    pub datalakes: Vec<TaskFormatted>,
+    /// tasks compatible with v2
+    pub modules: Vec<InputModule>,
+}
+
+/// Input module for the runner.
+#[serde_as]
```

Comment:
u define `InputModule` second time, can we move to some types.rs file?

Reply:
resolve with https://github.com/HerodotusDev/hdp/pull/86/commits/7197bc2594213b982f3c9a857c6d2481f05cf9d6